hassio/src/components/hassio-search-input.ts

```diff
@@ -0,0 +1,57 @@
+import { TemplateResult, html } from "lit-html";
+import { LitElement, CSSResult, css, property } from "lit-element";
+import "@polymer/iron-icon/iron-icon";
+import "@polymer/paper-input/paper-input";
+import "@material/mwc-button";
+
+class HassioSearchInput extends LitElement {
+  @property() private filter?: string;
+
+  protected render(): TemplateResult | void {
+    return html`
+      <div class="search-container">
+        <iron-icon icon="hassio:magnify"></iron-icon>
```

Comment:
Use the paper input support for prefixing icons instead. 

Same with making clear a suffix.

See docs https://www.webcomponents.org/element/@polymer/paper-input

Reply:
Added